include/grpc/event_engine/event_engine.h

```diff
@@ -261,6 +261,11 @@ class EventEngine {
     struct LookupTaskHandle {
       intptr_t key[2];
     };
+    /// Optional configuration for DNSResolvers.
+    struct ResolverOptions {
+      /// Override the DNSResolver's default authority.
```

Comment:
We should say something like "if empty, default DNS servers will be used."

Also, do we want to restrict this to being specified as IP:port, or do we want to allow EE impls to be able to support resolving this name using the default DNS servers?  Currently, we support only IP:port.

Reply:
Good question. I'm in favor of limiting it to IP:port for YAGNI reasons: it avoids the added complexity of supporting a feature that can't be used today. Changing gRPC's URI policy later would technically mean a breaking change for the engine API, but one that most applications will probably never encounter. WDYT?